RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Interface.swift

```diff
@@ -0,0 +1,47 @@
+import CloudKit
+import ComposableArchitecture
+import DependenciesAdditions
+import os
+
+// MARK: - CloudBackupClient
+public struct CloudBackupClient: DependencyKey, Sendable {
```

Comment:
The 3 files regarding `CloudBackupClient` are not part of the project, so I guess we don't need them on this PR?

Reply:
Sort of I guess, since they are never called. But the security center client reads the values that are written only here, so it made sense to put them in the same pr. but the files should also be added to the project for sure.